README.md

```diff
@@ -68,6 +68,13 @@ PuppetDB
   `Preferences>File Sharing` in order for these directories to be created
   and volume-mounted automatically. There is no need to add each sub directory.
 
+The default can be overridden with the `VOLUME_ROOT` environement
+variable in the `.env` file (see *Environment* below)
+
+```
+VOLUME_ROOT=/container_data/pupperware
+```
+
```

Comment:
I think moving this addition up to before the list of subdirectories would make more sense.